From 1c440f9420a520363d048d83840e5fad2e66dccd Mon Sep 17 00:00:00 2001
From: venkatesh <venkatesh.m@phytecembedded.in>
Date: Tue, 2 Apr 2024 11:16:03 +0530
Subject: [PATCH 1/2] Rb-a5d2x.dtsi


diff --git a/arch/arm/boot/dts/a5d2x-rugged_board_common.dtsi b/arch/arm/boot/dts/a5d2x-rugged_board_common.dtsi
index 7b854d501022..f25255273bae 100644
--- a/arch/arm/boot/dts/a5d2x-rugged_board_common.dtsi
+++ b/arch/arm/boot/dts/a5d2x-rugged_board_common.dtsi
@@ -274,7 +274,7 @@
 
 			flx4: flexcom@fc018000 {
 				atmel,flexcom-mode = <ATMEL_FLEXCOM_MODE_SPI>;
-				status = "disabled";
+				status = "okay";
 
 				uart6: serial@200 {
 					compatible = "atmel,at91sam9260-usart";
@@ -299,15 +299,22 @@
 					pinctrl-names = "default";
 					pinctrl-0 = <&pinctrl_mikrobus_spi &pinctrl_mikrobus1_spi_cs &pinctrl_mikrobus1_rst>;
 					atmel,fifo-size = <16>;
-					status = "disabled";
+					status = "okay";
 
 					spidev@0 {
 						compatible = "rohm,dh2228fv";
 						reg = <0>;
 						spi-max-frequency = <500000>;
-						status = "disabled";
+						status = "okay";
 					};
+
+					rc522@1 {
+						compatible = "phytec,rfid_rc522";
+						reg = <1>;
+						spi-max-frequency = <13560000>;
+						status = "okay";
 				};
+			};
 
 				i2c3: i2c@600 {
 					compatible = "atmel,sama5d2-i2c";
-- 
2.34.1

